components/financial_assistance/app/views/financial_assistance/applications/index.html.erb

```diff
@@ -7,6 +7,11 @@
       <div class="col-md-10">
         <h1 class="darkblue no-buffer"><%= l10n('faa.cost_savings_applications') %></h1>
         <h4><%= l10n('faa.cost_savings_applications_desc') %></h4>
+        <% if FinancialAssistanceRegistry.feature_enabled?(:iap_year_selection_form) %>
+          <% oe_year = HbxProfile.current_hbx.benefit_sponsorship.benefit_coverage_periods.last.open_enrollment_end_on.year %>
+          <% current_year = HbxProfile.current_hbx.benefit_sponsorship.benefit_coverage_periods.last.open_enrollment_start_on.year %>
+          <h4><b><%= l10n('faa.coverage_update_reminder', year: oe_year, year2: current_year) %></b></h4>
```

Comment:
@j1joey are the placement of the years correct? OE year will be the next year, while current year is this year.

Reply:
Updated the dates